src/nvim/options.lua

```diff
@@ -1909,13 +1909,6 @@ return {
       redraw={'current_window'},
       defaults={if_true=false}
     },
-    {
-      full_name='remap',
-      short_desc=N_("mappings to work recursively"),
-      type='bool', scope={'global'},
-      varname='p_remap',
-      defaults={if_true=true}
-    },
```

Comment:
Follow the pattern used for e.g. `'compatible'`: change `varname` to `'p_force_off'` and the descriptino to `"No description"`.